src/apiCalls.test.js

```diff
@@ -0,0 +1,163 @@
+import { fetchUser, getMovies,  getRatings, postRating } from './apiCalls';
+
+describe('apiCalls', () => {
+  describe('fetchUser', () => {
+    let mockEmail, mockPassword, mockUserInfo;
+
+    beforeEach(() => {
+      mockEmail = 'ken@turing.io';
+      mockPassword = '654321'
+      mockUserInfo = {email: mockEmail,password: mockPassword}
+      window.fetch = jest.fn().mockImplementation(() => {
+        return Promise.resolve({
+          ok: true,
+          json: () => Promise.resolve(mockUserInfo)
+        })
+      })
+    })
+
+    it.skip('should call fetch with url and options', () => {
```

Comment:
I couldn't get this particular test working for some reason.  Kept returning something that looked like 
`{email: {email: 'ken@turing.io, password: '654321} }`

Reply:
Interesting. I'll give it a look.